### ISSUE TITLE
Suggestion (Missions&Research)

### ISSUE THREAD

**Issue by TheOrangeGaming** [perfilrobo/hexbot#70]
If you could make the bot be able to do delete software missions or ddos missions that would be amazing and also if it could have like if you was on the fbi's most wanted and you was over a certain amount of $$ for bounty it could then change it's IP through the ISP?
and also a complete all missions button for when I want to go ahead and make a lot of money on a alt account and then transfer it over to my main. 
If you could also have a research bot that can automatically research a software that you want and you could set it to use a certain amount of $$ max and if it runs out of money in wallet it would either do missions or sell the amount of bitcoin that you need to research that software

**Comment by gresendesa** [comment]
Yeah. I've been being very busy lately, but I'll try to make the delete software missions sequence.

**Comment by TheOrangeGaming** [comment]
Any update?
